pkg/controller/wip/controller.go

```diff
@@ -342,7 +347,7 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 
 	servicePlan := findServicePlan(instance.Spec.PlanName, serviceClass.Plans)
 	if servicePlan == nil {
-		glog.Errorf("Instance %v/%v references a non-existent ServicePlan %v on ServiceClass %v", instance.Namespace, instance.Name, servicePlan.Name, serviceClass.Name)
+		glog.Errorf("Instance %v/%v references a non-existent ServicePlan %v on ServiceClass %v", instance.Namespace, instance.Name, instance.Spec.PlanName, serviceClass.Name)
```

Comment:
can you either wrap the %v's with quotes ('xx') or change some to %q so golang does it for you. Names can sometimes be words that make the sentence hard to parse for newbies because they blend into a normal english sentence.  Wrapping them with quotes will remove that risk.